f5_heat/resources/f5_ltm_pool.py

```diff
@@ -86,6 +86,7 @@ class F5LTMPool(resource.Resource, F5BigIPMixin):
         )
     }
 
+    @f5_common_resources
```

Comment:
Consider moving the definition of '`f5_common_resources`' into '`F5BigIPMixin`', and calling it as an instance method inside the method it is currently decorating, e.g.:
`self._f5_common_resources`.

Also, its name should probably start with '_'.